### PR TITLE
47335_Master layout & home for trainee

### PR DESCRIPTION
**Pull request by Omid07**
screenshot from 2015-12-02 15 06 14

### CODE REVIEW

File: app/Http/Middleware/User.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if ( \Auth::check() && \Auth::user()->isSupervisor()){
```
Comment (asifbuetcse) [inline app/Http/Middleware/User.php]: space before {

File: app/Http/Middleware/User.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            return redirect('/supervisor/home');
+        }else return redirect('/trainee/home');
```
Comment (asifbuetcse) [inline app/Http/Middleware/User.php]: space before }

File: public/css/stylesheet.css [hunk 3]
```diff
@@ -15,3 +15,6 @@
 .navbar-inverse .navbar-nav>li>a {
     color: #1F1C1C;
 }
+#profileRight{
```
Comment (asifbuetcse) [inline public/css/stylesheet.css]: space before {

File: app/Http/routes.php [hunk 4]
```diff
@@ -21,3 +21,10 @@
     Route::get('register', 'AuthController@getRegister');
     Route::post('register', 'AuthController@postRegister');
 });
+Route::get('/home', function () {
+	if(\Auth::User()->role){
+		return view('suoervisor.home');
```
Comment (yasin-buet) [inline app/Http/routes.php]: supervisor

File: app/User.php [hunk 5]
```diff
@@ -36,4 +36,5 @@ class User extends Model implements AuthenticatableContract,
      * @var array
      */
     protected $hidden = ['password', 'remember_token'];
+
```
Comment (yasin-buet) [inline app/User.php]: blank line remove

File: public/css/stylesheet.css [hunk 6]
```diff
@@ -15,3 +15,6 @@
 .navbar-inverse .navbar-nav>li>a {
     color: #1F1C1C;
 }
+#profileRight {
+left:1050px;
```
Comment (yasin-buet) [inline public/css/stylesheet.css]: space after :

File: resources/views/layouts/trainee.blade.php [hunk 7]
```diff
@@ -0,0 +1,46 @@
+<html>
```
Comment (dinhhoanglong91) [inline resources/views/layouts/trainee.blade.php]: Use `app.blade.php` as layout for this file

File: app/Http/routes.php [hunk 8]
```diff
@@ -21,3 +21,10 @@
     Route::get('register', 'AuthController@getRegister');
     Route::post('register', 'AuthController@postRegister');
 });
+Route::get('/home', function () {
+	if(\Auth::User()->role){
```
Comment (dinhhoanglong91) [inline app/Http/routes.php]: Coding convention

**Comment by yasin-buet** [comment]
OK

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**